### ISSUE TITLE
make it easier to navigate to highlighted text

### ISSUE THREAD

**Issue by jkldgoefgkljefogeg** [glowing-bear/glowing-bear#1016]
Click on red highlight badge to jump to highlighted message. It is very hard to find highlighted messages, especially when more lines need to be fetched

**Comment by lorenzhs** [comment]
There's a keyboard shortcut, Alt+A. They're all documented on the landing page. The discoverability isn't great, though...

**Comment by jkldgoefgkljefogeg** [comment]
You mean upper case A? Alt-a jumps jumps to next buffer with activity while unsetting notification. When I have hundreds of unread messages, it is extremely hard to navigate back to highlighted messages.

**Comment by lorenzhs** [comment]
Right, sorry, I totally misunderstood what you meant. That's true.

**Comment by xnox** [comment]
Regular weechat has the below commands.
It would be nice to have something similar.

Btw should /window command work?


Alt+n

Scroll to next highlight.

/window scroll_next_highlight

Alt+p

Scroll to previous highlight.

/window scroll_previous_highlight
